packages/node/test/integration/cross-node-multisig-creation.spec.ts

```diff
@@ -1,11 +1,22 @@
+import { Node as NodeTypes } from "@counterfactual/types";
 import { BaseProvider, JsonRpcProvider } from "ethers/providers";
 import { v4 as generateUUID } from "uuid";
 
-import { IMessagingService, IStoreService, Node, NodeConfig } from "../../src";
+import {
+  IMessagingService,
+  IStoreService,
+  Node,
+  NODE_EVENTS,
+  NodeConfig
+} from "../../src";
 import { MNEMONIC_PATH } from "../../src/signer";
```

Comment:
We're doing two kinds of path resolution here. `"../../src/signer"` and `"../../src"`; need to decide on one or the other I think